apps/web/playwright/organization/organization-invitation.e2e.ts

```diff
@@ -392,7 +392,7 @@ async function signupFromEmailInviteLink({
   const signupPage = await context.newPage();
 
   signupPage.goto(inviteLink);
-  await signupPage.waitForLoadState("networkidle");
+  await signupPage.locator(`[data-testid="signup-usernamefield"]`).waitFor({ state: "visible" });
```

Comment:
There is actually no network activity. We need to wait for username field to be visible before checking things.